sidecar/sidecar.go

```diff
@@ -110,8 +110,7 @@ func Run(conf config.Config) error {
 			return err
 		}
 
-		// TODO: Log failure?
-		go appSupervisor.DoLogManagement("/tmp/filebeat.yml")
+		appSupervisor.AddProcess([]string{"filebeat", "-c", "/tmp/filebeat.yml"}, []string{"GODEBUG=netdns=go"})
```

Comment:
Can we get rid of this completely?
Basically a YAML file format like this
```yaml
supervise:
 app: "cmd string with args"
 helpers:
   - helper cmd 1..
   - helper cmd 2..
```

We kill the container only if the app dies. if the helpers die, we don't do anything (for now). In future, we may want to restart it automatically.